Workbooks/UsageMetrics/Usage - Events/Usage - Events.workbook

```diff
@@ -435,7 +435,7 @@
             "name": "cohortUsersQuery",
             "type": 1,
             "description": "Extract query in cohort json from UsersList dropdown ",
-            "query": "let query = '{UsersList}';\r\n print parse_json(query).cohort",
+            "query": "let query = '{UsersList:escape}';\r\nprint parse_json(query).cohort",
```

Comment:
this is just formatting json then taking a field out?  you can't just use criteria param and jsonpath?
isn't this just `{UsersList}:$.cohort` ?

Reply:
(corrected my param syntax)

Reply:
Thanks. I will update it.